base_report_to_printer/models/ir_actions_report.py

```diff
@@ -110,7 +108,7 @@ def print_document(self, record_ids, data=None):
             raise exceptions.UserError(_("No printer configured to print this report."))
         if self.print_report_name:
             report_file_names = [
-                safe_eval(self.print_report_name, {"object": obj, "time": time})
```

Comment:
Why do you remove the safe_eval?

Reply:
I remove the safe_eval because the javascript does not work. Maybe it would have to be fixed for it to work well, but I didn't know how.

Reply:
Checking Odoo code, this is necessary. You can review it here:

https://github.com/odoo/odoo/blob/c9980b8f946005904cf4701b71a1c37e7577ea65/addons/web/controllers/main.py#L2032-L2034